src/main/java/com/learn/student/userdetails/Reportcardapi/Model/Student.java

```diff
@@ -0,0 +1,31 @@
+package com.learn.student.userdetails.Reportcardapi.Model;
+
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+
+@Entity
+@Table(name="students")
```

Comment:
Can you please change the table name cause we already have the table name for the student in the same database
and  can i know where the student data is coming from cause i havent seen any queries or table related to the code